sys/include/net/gcoap.h

```diff
@@ -446,6 +446,16 @@ extern "C" {
  */
 #define GCOAP_PAYLOAD_MARKER    (0xFF)
 
+/**
+ * @ingroup net_gcoap_conf
+ * @brief   Flag to delay initialization until explicit command
+ *
+ * By default gcoap starts at auto-initialization time.
+ */
+#ifndef GCOAP_INIT_DELAY
+#define GCOAP_INIT_DELAY        0
```

Comment:
All other macro values are surrounded by parantheses `(0)`, so to be uniform maybe here too?

Reply:
The parentheses defeat use of the new IS_ACTIVE macro.